inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/negative_layer_support_test.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+namespace LayerTestsDefinitions {
+
+#define GNA_UNSUPPPORTED_LAYER_NEG_TEST(test_class, expected_error_substring, test_configuration)       \
+class Negative##test_class : public test_class {                                                        \
+public:                                                                                                 \
+        void Run() override {                                                                           \
+                try {                                                                                   \
+                        test_class::LoadNetwork();                                                      \
+                        FAIL() << "GNA's unsupported layers were not detected during LoadNetwork()";    \
+                }                                                                                       \
+                catch (std::runtime_error& e) {                                                         \
```

Comment:
I think we need to catch only specific IE exceptions

Reply:
@nkogteva what's your opinion? Such construction of try block is used in [Convolutional negative tests](https://github.com/andreysapozhn/openvino/blob/master/inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/convolution_negative.cpp#L127-L137)